src/main/java/frc/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.TalonFXInvertType;
+
+public class ArmConstants {
+
+    // Arm constants
+    public static final double VOLT_COMP_SATURATION = 10; //[V]
+    public static final boolean ENABLE_VOLT_COMPENSATION = true;
```

Comment:
1. It makes more sense to first enable the voltage compensation, then set the specific value.
2. Don't shorten "voltage", use the full form.